apps/email/js/cards/compose.js

```diff
@@ -361,6 +361,18 @@ return [
       };
     },
 
+    haveAddressesChanged: function() {
+      var addrs = this.extractAddresses();
+      function compareAddresses(a, b) {
+        return a.length == b.length && !a.some(function(e, i) {
```

Comment:
Seems best to indicate what this function returns since it is the opposite of what `haveAddressesChanged` returns, maybe via a different function name.

Although thinking about it more, perhaps change the function to match the boolean state of `haveAddressesChanged`, so something like `addressesDiffer`, then this check becomes something like `a.length !== b.length || a.some(` where the function passed to some returns true if it detects a difference.

Then, the final return for `haveAddressesChanged` would be an `||` of those differ checks.